pkg/app/appcommon/log_store.go

```diff
@@ -4,16 +4,21 @@ import (
 	"bytes"
 	"fmt"
 	"io"
-	"os"
+	"regexp"
 	"strings"
 	"sync"
 	"time"
 
+	log "github.com/sirupsen/logrus"
 	"github.com/skycoin/skycoin/src/util/logging"
 	"go.etcd.io/bbolt"
 )
 
-const timeLayout = time.RFC3339Nano
+const (
+	timeLayout = time.RFC3339Nano
+)
+
+var re = regexp.MustCompile(`\x1b?\[[0-9;]*m`)
```

Comment:
Could you please use any library instead? For example, this one: https://github.com/acarl005/stripansi. The regular expression there could cover more cases.

Reply:
@nkryuchkov I think it better just copy their regular expression.